eclass/dlang-compilers-r1.eclass

```diff
@@ -0,0 +1,291 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: dlang-compilers-r1.eclass
+# @MAINTAINER:
+# Andrei Horodniceanu <a.horodniceanu@proton.me>
+# @AUTHOR:
+# Andrei Horodniceanu <a.horodniceanu@proton.me>
+# Based on dlang-compilers.eclass by Marco Leise <marco.leise@gmx.de>.
+# @BUGREPORTS:
+# Please report bugs via https://github.com/gentoo/dlang/issues
+# @VCSURL: https://github.com/gentoo/dlang
+# @BLURB: Support data for dlang-utils.eclass
+# @DESCRIPTION:
+# Contains the available D compiler versions with their stable
+# architectures and the language version they support.
+
+if [[ ${_DLANG_COMPILERS_R1_ECLASS} != "recur -_+^+_- spank" ]] ; then
+_DLANG_COMPILERS_R1_ECLASS="recur -_+^+_- spank"
```

Comment:
This is a vapierism. Just copy what modern day eclasses in ::gentoo do, please.

Reply:
I've left it in there as I think it's a funny string and it has some value as it's been in the eclass from the beginning. Alas, coding standards come before my sense of humor.